backend/src/test/java/wooteco/retrospective/presentation/AttendanceControllerTest.java

```diff
@@ -0,0 +1,129 @@
+package wooteco.retrospective.presentation;
+
+import static com.sun.org.apache.xalan.internal.xsltc.dom.LoadDocument.*;
+import static org.mockito.ArgumentMatchers.*;
+import static org.mockito.BDDMockito.*;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.*;
+import static org.springframework.test.web.servlet.result.MockMvcResultHandlers.*;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.*;
+import static org.springframework.restdocs.mockmvc.MockMvcRestDocumentation.document;
+
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.util.Collections;
+import java.util.List;
+import java.util.Optional;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.restdocs.AutoConfigureRestDocs;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.servlet.MockMvc;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import wooteco.retrospective.application.attendance.AttendanceService;
+import wooteco.retrospective.domain.attendance.Attendance;
+import wooteco.retrospective.domain.attendance.Time;
+import wooteco.retrospective.domain.member.Member;
+import wooteco.retrospective.infrastructure.dao.attendance.TimeDao;
+import wooteco.retrospective.presentation.attendance.AttendanceController;
+import wooteco.retrospective.presentation.dto.attendance.AttendanceRequest;
+
+@DisplayName("출석부 - Controller 테스트")
+@WebMvcTest(controllers = {AttendanceController.class})
```

Comment:
bean으로 등록할 컨트롤러가 하나라면
```suggestion
@WebMvcTest(AttendanceController.class)
```
이 형식이 좋아보이는데 이거 제가 이렇게 바꿨던거 같네요.? 죄송..